lib/actions/local/push.coffee

```diff
@@ -26,11 +26,12 @@ originalAction = balenaPush.action
 # TODO: This is a temporary workaround to reuse the existing `rdt push`
 # capitano frontend in `balena local push`.
 
+# coffeelint: disable-next-line ("Line ends with trailing whitespace")
 deprecationMsg = '''
-	------------------------------------------------------------------------------
+	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -  
 	Deprecation notice: `balena local push` is deprecated and will be removed in a
-	future release of the CLI. Please use `balena push <ipAddress>` instead.
-	------------------------------------------------------------------------------
+	future release of the CLI. Please use `balena push <ipAddress>` instead.  
+	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
* Replaced continuous dashes with dash-spaces because continuous dashes are interpreted by markdown as defining a title (the deprecation notice was being rendered in huge fonts!).
* Added some trailing double-spaces that markdown interprets as in-paragraph line breaks.
* Added coffeelint annotation to ignore said trailing spaces.